javascript-light-bulb/main.js

```diff
@@ -0,0 +1,20 @@
+// create storage for light bulb object
+// create storage for the background object
+// - if the light bulb is on
+//  - turn the light bulb off and make the background dark to match
+// - if it's off, turn the light bulb on and light up the background too
+
+var $lightBulb = document.querySelector('.light-bulb');
+var $container = document.querySelector('.container');
+
+function toggleSwitch(event) {
+  if ($lightBulb.className === 'light-bulb on-switch') {
```

Comment:
Instead of basing your functionality around the current classes on the DOM element, I would use a variable to keep track of if your lightbulb is "on" or "off". A boolean value would probably be a good choice. Remember what we discussed in Q&A, yesterday, we want all JavaScript functionality to not be reliant on data stored in the DOM as much as possible, you should have the information you need to do your work all centralized in your JavaScript.